### ISSUE TITLE
aws_vpc_peering_connection doesn't create tags

### ISSUE THREAD

**Issue by rcostanzo** [hashicorp/terraform#1724]
Using terraform 0.4.2

If you create an aws_vpc_peering_connection resource, the tags are never created. Also, the dependency graph seems a little off as the next run shows the delta that the tags need to be added. But then the run after that shows that the entire resource itself needs to be added. And then it finally says nothing needs to be done. Configuration and test output below:

```
variable "environment" {
    default = "vpc"
}

variable "peer_account_id" {
    default = "XXXX"
}

variable "peer_vpc_id" {
    default = "XXXX"
}

resource "aws_vpc" "extole" {
    cidr_block = "10.22.0.0/16"
    instance_tenancy = "default"
    enable_dns_support = true
    enable_dns_hostnames = true

    tags {
        Name = "extole-${var.environment}"
        environment = "${var.environment}"
    }
}

resource "aws_vpc_peering_connection" "extole" {
    peer_owner_id = "${var.peer_account_id}"
    peer_vpc_id = "${var.peer_vpc_id}"
    vpc_id = "${aws_vpc.extole.id}"

    tags {
        Name = "extole-yum-${var.environment}"
        environment = "${var.environment}"
    }
}
```

```
$terraform plan -out=myplan
Refreshing Terraform state prior to plan...
...
Path: terraform.tfplan.280415.193808

$terraform apply myplan
+ aws_vpc.extole
    cidr_block:                "" => "10.22.0.0/16"
    default_network_acl_id:    "" => "<computed>"
    default_security_group_id: "" => "<computed>"
    enable_dns_hostnames:      "" => "1"
    enable_dns_support:        "" => "1"
    instance_tenancy:          "" => "default"
    main_route_table_id:       "" => "<computed>"
    tags.#:                    "" => "2"
    tags.Name:                 "" => "extole-vpc"
    tags.environment:          "" => "vpc"

+ aws_vpc_peering_connection.extole
    peer_owner_id:    "" => "XXXX"
    peer_vpc_id:      "" => "XXXX"
    tags.#:           "" => "2"
    tags.Name:        "" => "extole-yum-vpc"
    tags.environment: "" => "vpc"
    vpc_id:           "" => "${aws_vpc.extole.id}"


Continue with apply? [y/N]y
aws_vpc.extole: Creating...
  cidr_block:                "" => "10.22.0.0/16"
  default_network_acl_id:    "" => "<computed>"
  default_security_group_id: "" => "<computed>"
  enable_dns_hostnames:      "" => "1"
  enable_dns_support:        "" => "1"
  instance_tenancy:          "" => "default"
  main_route_table_id:       "" => "<computed>"
  tags.#:                    "" => "2"
  tags.Name:                 "" => "extole-vpc"
  tags.environment:          "" => "vpc"
aws_vpc.extole: Creation complete
aws_vpc_peering_connection.extole: Creating...
  peer_owner_id:    "" => "XXXX"
  peer_vpc_id:      "" => "XXXX"
  tags.#:           "" => "2"
  tags.Name:        "" => "extole-yum-vpc"
  tags.environment: "" => "vpc"
  vpc_id:           "" => "XXXX"
aws_vpc_peering_connection.extole: Creation complete

Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
...
State path: terraform.tfstate


$terraform plan -out=myplan
Refreshing Terraform state prior to plan...

aws_vpc.extole: Refreshing state... (ID: XXXX)
aws_vpc_peering_connection.extole: Refreshing state... (ID: XXXX)
...
Path: terraform.tfplan.280415.193837

$terraform apply myplan
~ aws_vpc_peering_connection.extole
    tags.#:           "0" => "2"
    tags.Name:        "" => "extole-yum-vpc"
    tags.environment: "" => "vpc"


Continue with apply? [y/N]y
aws_vpc_peering_connection.extole: Modifying...
  tags.#:           "0" => "2"
  tags.Name:        "" => "extole-yum-vpc"
  tags.environment: "" => "vpc"
aws_vpc_peering_connection.extole: Modifications complete

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
...
State path: terraform.tfstate

$terraform plan -out=myplan
Refreshing Terraform state prior to plan...

aws_vpc.extole: Refreshing state... (ID: XXXX)
...
Path: terraform.tfplan.280415.193850

$terraform apply myplan
+ aws_vpc_peering_connection.extole
    peer_owner_id:    "" => "XXXX"
    peer_vpc_id:      "" => "XXXX"
    tags.#:           "" => "2"
    tags.Name:        "" => "extole-yum-vpc"
    tags.environment: "" => "vpc"
    vpc_id:           "" => "XXXX"


Continue with apply? [y/N]y
aws_vpc_peering_connection.extole: Creating...
  peer_owner_id:    "" => "XXXX"
  peer_vpc_id:      "" => "XXXX"
  tags.#:           "" => "2"
  tags.Name:        "" => "extole-yum-vpc"
  tags.environment: "" => "vpc"
  vpc_id:           "" => "XXXX"
aws_vpc_peering_connection.extole: Creation complete

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
...
State path: terraform.tfstate

$terraform plan -out=myplan
Refreshing Terraform state prior to plan...

aws_vpc.extole: Refreshing state... (ID: XXXX)
aws_vpc_peering_connection.extole: Refreshing state... (ID: XXXX)

No changes. Infrastructure is up-to-date. This means that Terraform
could not detect any differences between your configuration and
the real physical resources that exist. As a result, Terraform
doesn't need to do anything.
```

**Comment by johnrengelman** [comment]
I believe this is fixed in 0.5.0 - https://github.com/hashicorp/terraform/commit/c2fa397ec8835ecfb664b641320d470826393b38

**Comment by mitchellh** [comment]
Fixed
